src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -202,54 +208,52 @@ class GoogleDataprocDAO(protected val dataprocConfig: DataprocConfig, protected
   }
 
   /* Upload a file to a bucket as FileContent */
-  override def uploadToBucket(googleProject: GoogleProject, bucketName: String, fileName: String, content: File): Future[Unit] = {
+  override def uploadToBucket(googleProject: GoogleProject, bucketPath: GcsPath, content: File): Future[Unit] = {
```

Comment:
Just noting: once we do https://broadinstitute.atlassian.net/browse/GAWB-2625 we can probably remove this variant of `uploadToBucket` which takes a File.